template_cdk.json

```diff
@@ -32,7 +32,10 @@
         "enable_cw_rum":  "boolean_SET_CLOUDWATCH_RUM_APP_MONITOR|DEFAULT=false",
         "enable_cw_canaries": "boolean_SET_CLOUDWATCH_CANARIES_FOR_FRONTEND_TESTING|DEFAULT=false",
         "shared_dashboards_sessions": "string_TYPE_SESSION_SHARED_DASHBOARDS|(reader, anonymous) DEFAULT=anonymous",
-        "enable_opensearch_serverless": "boolean_USE_OPENSEARCH_SERVERLESS|DEFAULT=false"
+        "enable_quicksight_monitoring": "boolean_ENABLE_CONNECTION_QUICKSIGHT_RDS|DEFAULT=false",
+        "enable_opensearch_serverless": "boolean_USE_OPENSEARCH_SERVERLESS|DEFAULT=false",
+        "pivot_role_as_part_of_environment": "boolean_CREATE_PIVOT_ROLE_AS_PART_OF_ENVIRONMENT_STACK|DEFAULT=false",
```

Comment:
naming is hard :) it may be a personal opinion but I am not a big fan of this name, since it doesn't say anything if the pivot role is created automatically or manually, it just mentions that the pivot role is part of the environment stack

Did you consider other names for this parameter? For some params I see that we use a convention of 'enable_something'. Maybe we should think about a consistency here:
- "enable_pivot_role_auto_create"
or something else